src/app/shared/validation/validation.service.ts

```diff
@@ -6,9 +6,15 @@ export class ValidationService {
   private fieldValidationMessages = {
     usernameMinLength: 'Minimum length of username must be 6 characters.',
     usernameRequired: 'Username is required.',
+    skillNameRequired: 'Skill name is required.',
+    skillSchemaNameRequired: 'Skill schema name is required.',
     usernamePattern: 'Username must contain only alphabetic characters.',
     nameRequired: 'Name is required.',
     nameMaxLength: 'The maximum length of name is 50 characters.',
+    skillNameMaxLength: 'The maximum length of name is 100 characters.',
+    skillLevelDescription: 'The maximum length of description is 2000 characters.',
```

Comment:
the skill level description

Reply:
Hi @1ceb3rg fixed, Please check it again